mne/decoding/tests/test_receptive_field.py

```diff
@@ -147,4 +148,55 @@ def test_receptive_field():
     assert_raises(ValueError, ReceptiveField, tmin, tmax, 1, scoring='foo')
 
 
+@requires_sklearn_0_15
+def test_receptive_field_fast():
+    """Test that the fast solving works like Ridge."""
+    from sklearn.linear_model import Ridge
+    rng = np.random.RandomState(0)
+    y = np.zeros(500)
+    x = rng.randn(500)
+    for delay in range(-2, 3):
+        y.fill(0.)
+        slims = [(-4, 2)]
+        if delay == 0:
+            y = x.copy()
+        elif delay < 0:
+            y[-delay:] = x[:delay]
+            slims += [(-4, -1)]
+        else:
+            y[:-delay] = x[delay:]
+            slims += [(1, 2)]
+        for slim in slims:
+            for estimator in (Ridge(alpha=0.), 0.):
+                model = ReceptiveField(slim[0], slim[1], 1.,
+                                       estimator=estimator)
+                model.fit(x[:, np.newaxis], y)
+                assert_array_equal(model.delays_,
+                                   np.arange(slim[0], slim[1] + 1))
+                expected = (model.delays_ == delay).astype(float)
+                assert_allclose(model.coef_[0], expected, atol=1e-2)
+                p = model.predict(x[:, np.newaxis])[:, 0]
+                assert_allclose(y, p, atol=5e-2)
+    # multidimensional
+    x = rng.randn(1000, 3)
+    y = np.zeros((1000, 2))
+    slim = [-5, 0]
+    for ii in range(1, 5):
+        y[ii:, ii % 2] += (-1) ** ii * ii * x[:-ii, ii % 3]
```

Comment:
just some systematic and weird delay assignment -- the lines later in the test tell you what the result should look like